examples/remix/app/routes/buckets/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { ActionFunction, Form, useLoaderData } from "remix";
+import { s3Client } from "../../../backend/s3Client.server";
+import { createBucket, listBuckets } from "../../../backend/buckets.server";
+
+type BucketData = {
+    name: string;
+    createdAt: string;
+}
+
+type BucketsList = {
+    buckets: BucketData[];
+}
+
+// action is executed on the server on incoming POST requests
+export const action: ActionFunction = async ({ request }) => {
+    let { name } = Object.fromEntries(await request.formData());
```

Comment:
We should stick to const wherever we aren't in need of making an exception to do a reassignment, since that is something we'd wanna avoid.

FYI Reassignments leads to more cognitive processing when working with code, since variables then have multiple responsibilities